lib/libzfs/libzfs_import.c

```diff
@@ -248,6 +248,17 @@ is_mpath_whole_disk(const char *path)
 	return (wholedisk);
 }
 
+static int
+udev_device_is_ready(struct udev_device *dev)
+{
+#ifdef HAVE_LIBUDEV_UDEV_DEVICE_GET_IS_INITIALIZED
+	return (udev_device_get_is_initialized(dev));
+#else
+	/* wait for DEVLINKS property to be initialized */
+	return (udev_device_get_property_value(dev, "DEVLINKS") != NULL);
```

Comment:
OK, thanks for the explanation.  Since this bit of code is pretty clean either way I don't feel too strongly about this.  Since @don-brady effectively already did the work to optimize this case when `udev_device_get_is_initialized()` is available I suggest we leave it as is.  I don't see it causes us any long term maintainability issues which is my only real concern.